### ISSUE TITLE
Showing number of commits from last 1 year instead of the current year

### ISSUE THREAD

**Issue by Showrin** [anuraghazra/github-readme-stats#1053]
**Describe the bug**
The card is showing the number of commits from the last 1 year. Like: `Total Commits (2021): 1.5k`. As it's saying 2021 in the bracket, then shouldn't it either show the commits (620) from the year 2021 or show `Last 1 year` instead of `2021`?

**Expected behavior**
It should show the commit from the current year or change `Total Commits (2021)` to `Total Commits (Last 1 year)`.

**Screenshots / Live demo link (paste the github-readme-stats link as markdown image)**
![Screenshot 2021-05-11 at 1 22 03 PM](https://user-images.githubusercontent.com/62229369/117778537-8fda0780-b25f-11eb-8c5b-7dd96ae40fe3.png)

**Comment by florianbussmann** [comment]
Yes, the expected behavior is to show the contributions from last year and I support your opinion that this is misleading
https://github.com/anuraghazra/github-readme-stats/pull/211#issue-457030914

**Comment by rickstaa** [comment]
As pointed out in #1815 the current behaviour is misleading and should be fixed. As pointed out by @hashfx the card should have an option to choose between `year-wise`, `current year` or `total`. Further, the active mode should be displayed on the card. I am currently very short on time, so I can not implement this myself but feel free to create a PR for this feature to review. I added the ![](https://img.shields.io/github/labels/anuraghazra/github-readme-stats/popular) tag since I think this is an important bug that should be fixed.

**Comment by asifurrahaman754** [comment]
Hi is this sovled?

**Comment by adi-uchiha** [comment]
hey there, I want to work on this.
@rickstaa  Can you please assist me

**Comment by rickstaa** [comment]
@adi-uchiha This is still an issue. It would be great if this behaviour is improved. 💪🏻 I assigned you to this issue since I already assigned @asifurrahaman754 to several issues. Feel free to contact me if you have any questions.

**Comment by rickstaa** [comment]
@adi-uchiha, while checking #2107, I noticed that #1053 and #764 are related and that #1053 should best be solved in #2107 since this PR adds a lot of new code. Merging #1053 will therefore solve this issue. Sorry for the confusion.

**Comment by rickstaa** [comment]
This issue is solved when #2107 is merged.

**Comment by rickstaa** [comment]
Related to #764.

**Comment by alihamza1221** [comment]
is this issue solved ? If not can you assest me I am quite interested on working this issue

**Comment by qwerty541** [comment]
> is this issue solved ? If not can you assest me I am quite interested on working this issue

Hey, @alihamza1221! Thanks for your interest to the our project and considering contributions. Looks like we already have pull request https://github.com/anuraghazra/github-readme-stats/pull/2107 which will resolve this issue. Feel free to choose another issue that you are interested to work on.

**Comment by muskanyadav1134** [comment]
Can you please assign this to me?
